src/commands/request/request.ts

```diff
@@ -105,29 +116,93 @@ export async function runSasJob(
         return error
       }
 
-      let outputPath = path.join(
-        process.projectDir,
-        isLocal ? '/sasjsbuild' : ''
-      )
-
-      if (!(await folderExists(outputPath))) {
-        await createFolder(outputPath)
-      }
+      await writeOutput(outputPathParam, output, isLocal)
 
-      outputPath += '/output.json'
-
-      await createFile(outputPath, output)
       result = true
-      displaySuccess(`Request finished. Output is stored at '${outputPath}'`)
     })
-    .catch((err) => {
-      result = err
+    .catch(async (err) => {
+      let logData: string = ''
+
+      if (authConfig && err && err.logUrl) {
+        logData = JSON.stringify(
+          await fetchLogFileContent(
+            sasjs,
+            authConfig.access_token,
+            err.logUrl,
+            100000
+          )
+        )
+      } else {
+        if (err.error && err.error && err.error.details) {
+          logData = err.error.details.result
+        }
+
+        if (err.result) {
+          logData = err.result
+        }
+      }
+
+      if (jobPath) {
+        await saveLog(logData, logFile, jobPath, false)
```

Comment:
Can we [saveLog](https://github.com/sasjs/cli/blob/main/src/commands/flow/internal/saveLog.ts) from flow/internal.

If yes, we can extract that from internal to src/utils.

Reply:
Good point but I think that should be handled in a new issue since it's not quick fix.